app/models/cases.py

```diff
@@ -1,21 +1,22 @@
 from sqlmodel import Field, SQLModel
-from datetime import datetime
-from .categories import Categories
+from app.models.base import TableModelMixin
+from app.models.case_types import CaseTypes
 
 
 class BaseCase(SQLModel):
-    category: Categories = Field(index=True)
-    name: str
+    case_type: CaseTypes = Field(
+        index=True
+    )  # Which service is the case originally from
+    assigned_to: str | None = Field(
```

Comment:
we have removed 'assigned_to' to a seperate table - sorry!

Reply:
Thank you! I have removed assigned_to from this model with the latest commit!